dotcom-rendering/fixtures/manual/key-events.ts

```diff
@@ -14,7 +14,7 @@ const format = {
 	theme: ArticlePillar.News,
 };
 
-export const events: KeyEventCard[] = [
+export const events: [KeyEventCard, ...KeyEventCard[]] = [
```

Comment:
Does this change convince TS there is always atleast 1 item in this array?

Reply:
Is there no world where TS understands a statically declared array will always have at least as many elements as its originally defined with?
Or does the existence of modify-in-place function like Array.splice ruin this for us?

Reply:
You could use `as const`, but then you run into other kinds of issues where you’re passing on `readonly` types to mutable types.

I think this could be a worthwile endeavour, but I think we should discuss as a team the various options.

Reply:
Here’s a propsal: https://github.com/guardian/dotcom-rendering/pull/5817/files